api/base/views.py

```diff
@@ -31,6 +31,8 @@
 
 class JSONAPIBaseView(generics.GenericAPIView):
 
+    ordering = ('-date_modified',)
```

Comment:
The base view class should not make assumptions about what model fields exist on the models; that's the subclasses' responsibility. So I think this can be removed.